veascan-web/src/components/ColoredLabel.tsx

```diff
@@ -0,0 +1,37 @@
+import { Tag } from "@kleros/ui-components-library";
+import React from "react";
+import styled from "styled-components";
+
+interface ColoredLabelProps {
+  variant: string;
```

Comment:
Could you make the variant typing more specific? That way we would get autocompletion on the variants, and errors if we miss-type.
```typescript
variant: keyof typeof variantColors;
```